### DISCUSSION TITLE
Domains are configured in a team that I no longer have access to

### DISCUSSION THREAD

**Discussion by glowdragon** [vercel/community#5376] [Help]
### Summary

I just have noticed that all my domains are missing from my Vercel account, even though they are still attached to my personal projects that are hosted on your platform.

The missing domains include:
- creamcheese.games
- creamcheesegames.com
- danielkreitsch.com
- danielkreitsch.de
- eyeaeh.de
- glowdragon.de
- glowdragon.dev
- hytaleserver.io
- sailsofwonder.com
- sailsofwonder.de

I am unsure whether this issue is the result of a display bug or a configuration error. Therefore, I would greatly appreciate your guidance on how to resolve this situation.

### Example

https://vercel.com/glowdragon/~/domains

### Steps to Reproduce

1. Login to Vercel
2. Click on "Domains"
3. Find the message "No domains to show"

**Comment by z4nr34l** [comment]
I can confirm that domains added to any project are listed on `Domains` tab. This can be related to Vercel's DNS and registrar service as I can see there (on older team) only domains that were added before they launched that service.
